config/localization/en/app.json

```diff
@@ -633,9 +633,17 @@
       "WITHDRAWAL_ALERT_SUBTITLE": "Check status",
       "TRANSFEROUT_ALERT_TITLE": "You have pending transfer of {PENDING_TRANSFERS} {TOKEN}.",
       "TRANSFEROUT_ALERT_SUBTITLE": "Check status",
+      "WALLET_RESTRICTED_ERROR_TITLE": "Wallet Restricted",
+      "WALLET_RESTRICTED_ERROR_MESSAGE": "Because this address appears to be a resident of, or trading from, a jurisdiction that violates our terms of use, or has engaged in activity that violates our terms of use, this address has been blocked.",
+      "WALLET_RESTRICTED_WITHDRAWAL_TRANSFER_DESTINATION_ERROR_MESSAGE": "Because the destination address appears to be a resident of, or trading from, a jurisdiction that violates our terms of use, or has engaged in activity that violates our terms of use, the destination address has been blocked and this transaction cannot be completed.",
+      "WALLET_RESTRICTED_WITHDRAWAL_TRANSFER_ORIGINATION_ERROR_MESSAGE": "Because the origination address appears to be a resident of, or trading from, a jurisdiction that violates our terms of use, or has engaged in activity that violates our terms of use, the origination address has been blocked and this transaction cannot be completed.",
+      "RATE_LIMIT_REACHED_ERROR_TITLE": "Rate Limit Reached",
+      "RATE_LIMIT_REACHED_ERROR_MESSAGE": "Rate limited reached for this IP address. Please try again later.",
```

Comment:
Could we move out of `APP.ONBOARDING` object into `ERRORS.ONBOARDING` object?
The two `RATE_LIMIT` strings could go into `ERRORS.GENERAL`